lib/publiccloud/instance.pm

```diff
@@ -355,7 +355,7 @@ sub wait_for_ssh {
     # DMS will return normal user and error will be resolved: connection retry for that error.
 
     $args{username} //= $self->username();
-    my $delay = 1;
+    my $delay = 1;    # delay increased in loop after retrying longer than 2min
```

Comment:
```suggestion
    my $delay = $args{timeout} > 120 ? 5 : 1;    # delay increased in loop after retrying longer than 2min
```

let's not over complicate it , and you can remove line in the loop

Reply:
But this way statically predefined, a delay=5 always will be applied, having the most cases timeout > 120, now it is 600.
 
Indeed I used more code to change value, but only in those cases where really the test is taking too much time to connect.

Reply:
yes I think it will be better . 1 second is certainly too fast

Reply:
OK, to shorten, we could keep fast retrying 1s the only tests within 3min: `... = $args{timeout} > 180 ? 5 : 1;`

Reply:
done